syntax/forth.vim

```diff
@@ -195,11 +195,15 @@ syn match forthFloat '\<-\=\d*[.]\=\d\+[DdEe][-+]\d\+\>'
 syn region forthComment start='0 \[if\]' end='\[endif\]' end='\[then\]' contains=forthTodo
 
 " Strings
-syn region forthString start=+\.*\"+ end=+"+ end=+$+ contains=@Spell
+syn region forthString start=+\.*"+ end=+"+ end=+$+ contains=@Spell
 " XXX
-syn region forthString start=+s\"+ end=+"+ end=+$+ contains=@Spell
-syn region forthString start=+s\\\"+ end=+"+ end=+$+ contains=@Spell
-syn region forthString start=+c\"+ end=+"+ end=+$+ contains=@Spell
+syn region forthString start=+s"+ end=+"+ end=+$+ contains=@Spell
+syn region forthString start=+c"+ end=+"+ end=+$+ contains=@Spell
```

Comment:
I wonder if i bumped into some kind of bug now. If I open a Forth buffer, enter `s" "` or `c" "`, then activate syntax highlighting, the `s` or `c` is highlighted differently. Do you get the same?

Reply:
...it appears that it is a SpellCap problem. If I change `s" "` to `S" "`, the highlighting disappears.

Reply:
Would you agree that adding `setlocal spellcapcheck=` to forth.vim would be a good fix?

Reply:
Brilliant! That worked, thank you!